test/scripts/measures/2018/import-qcdr-measures-spec.js

```diff
@@ -0,0 +1,62 @@
+// Libraries
```

Comment:
Should this file be named `import-quality-measures-spec.js`?

Reply:
duplicate comment, yup good catch